### PR TITLE
[PWD-2658] - Add starting from on fixed program cards

### PR DESCRIPTION
**Pull request by kevintanes**
### What type of PR is this? (_check all applicable_)

- [ ] Feature
- [x] Bug Fix
- [ ] Documentation Update
- [ ] Setup
- [ ] Code Refactor
- [ ] Performance Improvements
- [ ] Test
- [ ] Build
- [ ] CI
- [ ] Revert

### Screenshots (_optional_)

video learnings 
image

fixed program
SAB: 
image

JC: 
Bahasa
image
english
image

### CODE REVIEW

**Review by Vesals — APPROVED** [review]

File: src/components/ProgramCard.tsx [hunk 1]
```diff
@@ -147,26 +147,27 @@ const ProgramCard: FC<ProgramCardProps> = ({
...
+                ? 'Starting From'
+                : 'Mulai Dari'}
+            </Typography>
+
+            <Typography
+              typography="subheading2"
+              color="warning100.main"
+              minHeight="26px"
+            >
+              {!program.ctPrice ? '-' : program.ctPrice}
```
Comment (Vesals) [inline src/components/ProgramCard.tsx:167]: atau kalo mau hemat 1 karakter 🐧 
```suggestion
              {program.ctPrice ? program.ctPrice : '-' }
```

Comment (shadiqfauzi) [inline src/components/ProgramCard.tsx:167]:
```suggestion
              {!program?.ctPrice ? '-' : program?.ctPrice}
```
lebih aman 😄

File: src/components/ProgramCard.tsx [hunk 2]
```diff
@@ -147,26 +147,27 @@ const ProgramCard: FC<ProgramCardProps> = ({
...
-
-          <Typography
-            typography="subheading2"
-            color="warning100.main"
-            minHeight="26px"
-          >
-            {!program.ctPrice ? '-' : program.ctPrice}
-          </Typography>
-        </Stack>
+        {isVoucherRedeem ? (
```
Comment (shadiqfauzi) [inline src/components/ProgramCard.tsx:150]:
```suggestion
        {isVoucherRedeem && <Component />}
```
kalau ada render condition dan kalau false ga render apa2, bisa gini

File: src/components/ProgramCard.tsx (hunk 1 above)
Comment (kevintanes) [reply src/components/ProgramCard.tsx:167]: okee kakk

File: src/components/ProgramCard.tsx (hunk 2 above)
Comment (kevintanes) [reply src/components/ProgramCard.tsx:150]: ohh. okeokee. tak ubahh dlu

**Review by shadiqfauzi — APPROVED** [review]

**Review by Vesals — APPROVED** [review]

**Pull request merged**